lib/message.rb

```diff
@@ -0,0 +1,16 @@
+require 'pry'
+
+class Message
+
+  attr_reader :text, :translator
```

Comment:
Be sure that you're only creating `attr_reader`s for things that you want to have access to from outside of your class. I like that you took both of the arguments that are passed to `initialize` and saved them as instance variables (need access to them in `translation` without passing anything as an argument), but that doesn't automatically mean that you need an `attr_reader`.